web-src/src/common/components/history/executions-log-table.vue

```diff
@@ -44,38 +62,64 @@ export default {
     }
   },
 
+  data() {
+    return {
+      filteredRows: this.rows ? [...this.rows] : [],
+      searchText: '',
+      mySortColumn: this.sortColumn,
+      myAscending: this.ascending
+    }
+  },
+
+  watch: {
+    rows: function(val, oldVal) {
+      this.searchText = '';
```

Comment:
I think this is not needed. If new data is loaded, I don't see any reason to reset search string.

Reply:
Sure. This behaviour is also fine.